tools/check_submodules/Cargo.toml

```diff
@@ -12,4 +12,4 @@ git2 = "0.7"
 hex = "0.3"
 log = "0.4"
 
-edgelet-utils = { path = "../../edgelet/edgelet-utils" }
+edgelet-utils = { path = "../../edgelet/edgelet-utils" }
```

Comment:
Same here, the file in master has a trailing newline.

Reply:
The missing newline characters have returned.